src/lib/queries.e2e.tsx

```diff
@@ -236,4 +237,19 @@ describe("React Query Hook Tests for RPC Endpoints", () => {
       `);
     });
   });
+
+  describe("ibcDenomsQuery Query", () => {
+    it("should return data", async ({ api, wrapper }: QueryTestContext) => {
+      const { result, waitFor } = renderHook(
+        () => useQuery(ibcDenomsQuery(api)),
+        { wrapper }
+      );
+
+      await waitFor(() => result.current.isSuccess);
+      expect(result.current.data).toBeDefined();
+      // XXX why does agoric-3-proposals not implement this endpoint?
+      // should be [{ "path": "transfer/channel-0", "base_denom": "uatom"}]
+      expect(result.current.data).toMatchInlineSnapshot("[]");
```

Comment:
The ATOM in `agoric-3-proposals` didn't actually come over IBC. It was manually added to some accounts at genesis.
https://github.com/Agoric/agoric-3-proposals/blob/ab2422a8c8735f6d678413c7edf6c086cc5d0e2b/upgrade-test-scripts/start_ag0.sh#L49

This query does work on mainnet:

```
agd query ibc-transfer denom-traces --node https://main.rpc.agoric.net:443
denom_traces:
- base_denom: uatom
  path: transfer/channel-0
- base_denom: factory/osmo1mlng7pz4pnyxtpq0akfwall37czyk9lukaucsrn30ameplhhshtqdvfm5c/ulvn
  path: transfer/channel-1
- base_denom: uatom
  path: transfer/channel-1/transfer/channel-0
- base_denom: inj
  path: transfer/channel-1/transfer/channel-122
...
```

and devnet:
```
$ agd query ibc-transfer denom-traces --node https://devnet.rpc.agoric.net:443
denom_traces:
- base_denom: "79228162514264337593543950342"
  path: transfer/channel-0/transfer/channel-2/transfer/channel-15
- base_denom: ubld
  path: transfer/channel-1
- base_denom: ubld
  path: transfer/channel-10
- base_denom: ubld
  path: transfer/channel-2
```

Reply:
Thanks, this makes sense. Would still expect _something_ from this endpoint in a3p. An empty array is surprising.